base_import_async/models/base_import_async.py

```diff
@@ -61,8 +61,8 @@ def _create_csv_attachment(session, fields, data, options, file_name):
     # write csv
     f = StringIO()
     writer = csv.writer(f,
-                        delimiter=options.get(OPT_SEPARATOR),
-                        quotechar=options.get(OPT_QUOTING))
+                        delimiter=str(options.get(OPT_SEPARATOR)),
+                        quotechar=str(options.get(OPT_QUOTING)))
```

Comment:
@guewen could be. It's not been an issue in practice so far. I guess it would impact only people using the API, which is kind of undocumented anyway.